chart/compass/values.yaml

```diff
@@ -636,6 +636,7 @@ global:
       maxIdleConnections: 2
     globalRegistryUrl: http://compass-external-services-mock.compass-system.svc.cluster.local:8087/.well-known/open-resource-discovery
     maxParallelApplicationProcessors: 4
+    containerName: "ord-aggregator"
```

Comment:
I see you've added this config similarly how it is done in the System Fetcher test. Make sure to use this config value in the actual `ord-aggregator-job.yaml` as the name of the container (similarly to `system-fetcher-job.yaml` in the sys fetcher PR that you're looking from) otherwise it's practically useless to define it